docs/TranslationGuide.md

```diff
@@ -4,9 +4,9 @@
 
 - Version 0.4.0
   - Added:
-    - `settingsVisContoursRainbowRepeatRate`, `settingsVisContoursRainbowIntensity`, `sceneExplorerProjectChangeFailedWrite`, `sceneExplorerProjectChangeFailedRead`, `sceneExplorerProjectChangeFailedParse`, `convertAllSpaceLabel`, `loginEnterURL`
+    - `settingsVisContoursRainbowRepeatRate`, `settingsVisContoursRainbowIntensity`, `sceneExplorerProjectChangeFailedWrite`, `sceneExplorerProjectChangeFailedRead`, `sceneExplorerProjectChangeFailedParse`, `convertAllSpaceLabel`, `loginEnterURL`, `settingsControlsMouseInvertX`, `settingsControlsMouseInvertY`, `settingsControlsControllerInvertX`, `settingsControlsControllerInvertY`
   - Changed:
-    - ?
+    - `settingsControlsInvertX`, `settingsControlsInvertY`
```

Comment:
I'm pretty sure the `Changed` section is for when you change the value of the key, but in this case you've changed the key too, so it should be in the removed section. @OsiJr is this correct?

Reply:
Yeah, technically these are removed/added keys.